app/models/course/assessment/submission.rb

```diff
@@ -135,6 +138,16 @@ def publish(_ = nil)
     end
   end
 
+  # Handles the unsubmission of a submitted submission.
+  #
+  # This will revert all manually graded answers to an attempting state.
```

Comment:
@weiqingtoh Sorry just to check, why only manually graded answers are reverted ?